aria/appilcation_tools.py

```diff
@@ -0,0 +1,257 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+import json
+import shutil
+import tarfile
+import tempfile
+from copy import copy
+from datetime import datetime
+
+from aria.logger import LoggerMixin
+from aria.exceptions import StorageError
+
+
+class StorageManager(LoggerMixin):
+    def __init__(
+            self,
+            model_storage,
+            resource_storage,
+            blueprint_path,
+            blueprint_id,
+            blueprint_plan,
+            deployment_id,
+            deployment_plan,
+            *args,
+            **kwargs):
+        super(StorageManager, self).__init__(*args, **kwargs)
+        self.model_storage = model_storage
+        self.resource_storage = resource_storage
+        self.blueprint_path = blueprint_path
+        self.blueprint_id = blueprint_id
+        self.blueprint_plan = blueprint_plan
+        self.deployment_id = deployment_id
+        self.deployment_plan = deployment_plan
+
+    @classmethod
+    def from_blueprint(
+            cls,
+            model_storage,
+            resource_storage,
+            blueprint_path,
+            blueprint_id):
+        return cls(
+            model_storage,
+            resource_storage,
+            blueprint_path,
+            blueprint_id,
+            deployment_id=None,
+            deployment_plan=None)
+
```

Comment:
let's do another classmethod for deployment
"
def from_deployment(cls, model_storage, resource_storage, deployment_plan, ...
"